_posts/2022-03-15-kubeflow-1.5-release.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Kubeflow v1.5 improves ML model accuracy, reduces infrastructure costs and optimizes MLOps"
+layout: post
+toc: true
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-1.5-release/
+author: "Josh Bottum, Kimonas Sotirchos, Thea Lamkin"
```

Comment:
I think that instead of writting names of authors, the author should be the v1.5 release team similar to what kubernetes does https://kubernetes.io/blog/2021/12/07/kubernetes-1-23-release-announcement/

Reply:
I have proposed this change